src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -448,15 +448,22 @@ export function MakeEnrichmentsTabUI(
             } else {
                 const content: any = [];
                 content.push(
-                    <OverlapExclusionIndicator
-                        store={store}
-                        only={
-                            patientAnalysisPossible &&
-                            store.usePatientLevelEnrichments
-                                ? 'patient'
-                                : 'sample'
-                        }
-                    />
+                    <div
+                        style={{
+                            marginLeft:
+                                enrichmentType == 'alterations' ? 213 : 0,
```

Comment:
I added:

> The alteration type selector is shown to left of the
graph panels ('in-line'). This div element pushes the
graph elements to the right when the type selector
is shown 'in-line'.

This comment will be extended in the next _driver-filtering_ PR (when the menu also can be show as a button depending on available data.)

Reply:
why do we need to bump that over?  this is kind yucky.

Reply:
This is because the type selection menu should be shown when the data is loaded. When the menu is in the DOM of the enrichmentContainer it will disappear and reload after each update of selected alteration types. The solution I chose was to put the menu as an overlay and add a bump to make place in the EnrichmentContainer.